app/views/rulesets/index.html.erb

```diff
@@ -11,8 +11,14 @@
             </h1>
           </legend>
           <div class="govuk-radios" data-module="govuk-radios">
+            <% Linters::Rulesets::ACTIVE_RULESETS.each do |ruleset_name| %>
+              <div class="govuk-radios__item">
+                <%= text_field_tag("ruleset_name", ruleset_name, class: "govuk-radios__input", type: "radio", name: "ruleset_name", checked: true) %>
+                <%= label_tag("gov-ruleset", t("rulesets.#{ruleset_name}.description"), class: "govuk-label govuk-radios__label", for: "gov-ruleset") %>
+              </div>
+            <% end %>
             <div class="govuk-radios__item">
-              <%= text_field_tag("ruleset_name", "government_ruleset", class: "govuk-radios__input", type: "radio", name: "ruleset_name", checked: true) %>
+              <%= text_field_tag("ruleset_name", "spectral", class: "govuk-radios__input", type: "radio", name: "ruleset_name", checked: true) %>
               <%= label_tag("gov-ruleset", "UK government ruleset", class: "govuk-label govuk-radios__label", for: "gov-ruleset") %>
```

Comment:
[change]

I think the `for` here will be `"ruleset_name_spectral"`.

Reply:
I've since removed this bit of code - the 3 dynamic rulesets replace the hardcoded government ruleset.

Reply:
I believe line 22 should be:
```ruby
 <%= label_tag("gov-ruleset", "UK government ruleset", class: "govuk-label govuk-radios__label", for: "ruleset_name_spectral") %>
```
But again, fire up the page and check the id there.